NevinChen.md

```diff
@@ -1,6 +1,24 @@
+
+## 10/31 ~ 11/04
+
+- [WIP] [Bug 1029649](https://bugzilla.mozilla.org/show_bug.cgi?id=1029649) -  [meta] Locale switching functionality for RTL
+    - Survied and made plans for action items
```

Comment:
We don't generally assigned people to a [meta] bug (meta means a bug that is used to sort actions required for a given feature/project) unless you want to drive the project on your own. I don't really know the scope of RTL (and if all the issues has been filed already) so I don't really know if you could single handed own this feature. If *not*, please unassign this bug or assign it to Vance, and assign yourself to the actual bugs.